File: src/todolist.py
class Todolist:

    def __init__(self):

        self.hari = {
            "senin":(),
            "selasa:":(),
            "Rabu":(),
            "Kamis":(),
            "Jumat":(),
            "Sabtu":(),
            "Minggu":()
        }
    def terima_agenda(self):
        for i in list(self.hari.keys()):
            agenda = input(f'masukan agenda anda hari {i} dipisahkan koma:  ')
            agenda = set(agenda.split(","))
            self.hari[i] = agenda

    def tulis(self):
        with open('src/text.txt',"a") as buka:
            for i in  list(self.hari.keys()):
                print(f'hari {i} ini agenda anda = {self.hari[i]}', file = buka) 
            print('\n', 59*'-', '\n',file=buka)
            
def main():
    instance = Todolist()
    instance.terima_agenda()
    instance.tulis()
    
main()
    

File: src/todolist.cpp
#include "todolist.h"

void todolist::terima()
{
    for(int i = 0;i<7;i++)
    {
        std::cout<<"Masukan Agenda Anda Pada Hari "<<hari[i]<<" :";
        std::getline(std::cin,kegiatan);
        dict.insert({hari[i],kegiatan});
    }

}

void todolist::tulis()
{
    std::fstream fw("src/textcpp.txt", std::ofstream::out);
    std::map<std::string, std::string>::iterator it = dict.begin();

    if (fw.is_open())
    {
        while (it != dict.end())
        {
            fw << "agenda anda pada hari "<<it->first<<" adalah: "<<it->second<< "\n";
            it ++;
        }
        fw<<"\n";
        fw.close();
    }
    else std::cout << "\nProblem with opening file";
}